src/libcore/mem/maybe_uninit.rs

```diff
@@ -18,10 +18,7 @@ use crate::mem::ManuallyDrop;
 ///
 /// ```rust,no_run
 /// # #![allow(invalid_value)]
-/// use std::mem::{self, MaybeUninit};
-///
-/// let x: &i32 = unsafe { mem::zeroed() }; // undefined behavior!
```

Comment:
Why did you remove this?

Reply:
While it hasn't been depreciated like `mem::uninitialized` has been, IIUC `mem::zeroed` is fairly niche as it's "useful for FFI sometimes, but should generally be avoided.". So I didn't think it needed to be front and center in the docs.

Reply:
`mem::zeroed` has some benefits over `MaybeUninit::zeroed().assume_init()` though: it is shorter, and once https://github.com/rust-lang/rust/pull/66059 lands it will have better dynamic checks. I am not sure that it is more niche then `MaybeUninit::zeroed()`?